src/Paket.Core/PackageManagement/NugetConvert.fs

```diff
@@ -28,19 +28,19 @@ type CredsMigrationMode =
 
     static member ToAuthentication mode sourceName auth =
         match mode, auth with
-        | Encrypt, Credentials(username, password, authType) -> 
-            ConfigAuthentication(username, password, authType)
-        | Plaintext, Credentials(username, password, authType) -> 
-            PlainTextAuthentication(username, password, authType)
```

Comment:
This change is a bit fishy. I have no idea where this information was used (or how it was supposed to be used). Apparently this information was not used at all or I introduced a bug.